Vonk.Plugin.ExampleOperation/CustomContentTypeMiddleware.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Vonk.Core.Support;
+
+namespace Vonk.Plugin.ExampleOperation
+{
+    internal class CustomContentTypeMiddleware
+    {
+        private readonly RequestDelegate _next;
+        private readonly ILogger<VonkPluginService> _logger;
+
+        private const string _exampleContentType = "ExampleContentType";
+
+        public CustomContentTypeMiddleware(RequestDelegate next, ILogger<VonkPluginService> logger)
+        {
+            Check.NotNull(next, nameof(next));
+            Check.NotNull(logger, nameof(logger));
+            _next = next;
+            _logger = logger;
+        }
+
+        // VonkPluginMiddleware can directly access the HttpContext, as opposed to the VonkPluginService
```

Comment:
"VonkPluginMiddleware" and "VonkPluginService" are no defined terms. Suggestion: A plugin implemented directly as ASP.NET Core Middleware can access the HttpContext directly, whereas a Service acting on the IVonkContext cannot.